google/resource_compute_global_address_test.go

```diff
@@ -29,6 +31,29 @@ func TestAccComputeGlobalAddress_basic(t *testing.T) {
 	})
 }
 
+func TestAccComputeGlobalAddress_ipv6(t *testing.T) {
+	var addr computeBeta.Address
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckComputeGlobalAddressDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccComputeGlobalAddress_ipv6,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckComputeBetaGlobalAddressExists(
```

Comment:
Consider asserting on the format of the computed address to make sure an IPv6 is used.
Additionally, to prevent regression, make sure the ipv4 address follows the ipv4 pattern.